src/main/webapp/app/app.tsx

```diff
@@ -62,12 +65,17 @@ export const App = (props: IAppProps) => {
                 />
               </ErrorBoundary>
             </View>
-            <View padding={'size-150'} gridArea="content" flex={true} alignSelf={'stretch'}>
+            <View
+              gridArea="content"
+              flex={true} alignSelf={'stretch'}
+              backgroundColor="static-gray-200">
```

Comment:
```suggestion
              backgroundColor="primary">
```
The above can be done, but we can make this change as part of the home page implementation